index.html

```diff
@@ -0,0 +1,121 @@
+            </div>
+          </div>
+  </header>
+  <main class="container">
+    <div class="programs" >
+      <h2 class="title">Main Program</h2>
+      <hr class="red_bar">
+      <div class="programs_cards" id="programs_cards">
+            <div class="card_program">
+              <img src="/pictures/program_icon_01.png" alt="lecture_icon">
+              <h3>Lecture</h3>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum animi libero similique dolor?</p>
+            </div>
+            <div class="card_program">
+              <img src="/pictures/program_icon_02.png" alt="lecture_icon">
+              <h3>NC Exhibition</h3>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum animi libero similique dolor?</p>
+            </div>
+            <div class="card_program">
+              <img src="/pictures/program_icon_03.png" alt="lecture_icon">
+              <h3>Forum</h3>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum animi libero similique dolor?</p>
+            </div>
+            <div class="card_program">
+              <img src="/pictures/program_icon_04.png" alt="lecture_icon">
+              <h3>Workshop</h3>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum animi libero similique dolor?</p>
+            </div>
+            <div class="card_program">
+              <img src="/pictures/program_icon_05.png" alt="lecture_icon">
+              <h3>NC Ignite</h3>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum animi libero similique dolor?</p>
+            </div>
+      </div>
+      <div class="action">
+        <a href="" class="desktop_seemore wholeProgram">SEE THE WHOLE PROGRAM</a>
+          <button class="join more_mobile">Join NC Global Summit 2015</button>
+      </div>
+    
+    </div>
+    <div class="speakers" >
+      <h2 class="title">Featured Speakers</h2>
+      <hr class="red_bar2">
+      <div class="speakers_cards" id="speakers_cards">
+      </div>
+                <button class="more_mobile more">MORE<i class="fa-solid fa-angle-down downIcon"></i></button>
+    </div>
+    <div class="partners desktop_seemore">
+      <h3>Partner</h3>
+      <hr class="red_bar">
+      <ul class="partener_logos">
+        <li><img src="/pictures/jackd_624-logo,.png" alt="logo_jackDaniel" srcset=""></li>
+        <li><img src="/pictures/budapest-logo.png" alt="logo_budapest" srcset=""></li>
+        <li><img src="/pictures/cocacola2-logo.png" alt="logo_Cocacola" srcset=""></li>
+        <li><img src="/pictures/disney-logo.png" alt="logo_disney" srcset=""></li>
+        <li><img src="/pictures/dreher-logo.png" alt="logo_dreher" srcset=""></li>
+      </ul>
+    </div>
+  </main>
+  <footer class="desktop_seemore">
+      <img class="logoLionFooter" src="/pictures/logoLion.png" alt="">
```

Comment:
Kindly add alt element to all your img elements. These will increase the accessibility of your web page and help to people with visual impairments and using special technologies to surf on your website.